dashboards/grafana-dashboard-insights-inventory-general.configmap.yaml

```
@@ -1010,6 +1010,97 @@ data:
           ],
           "valueName": "current"
         },
+        {
+            "cacheTimeout": null,
+            "colorBackground": true,
+            "colorPostfix": false,
+            "colorValue": false,
+            "colors": [
+              "#d44a3a",
+              "rgba(237, 129, 40, 0.89)",
+              "#299c46"
+            ],
+            "datasource": "$datasource",
+            "fieldConfig": {
+              "defaults": {
+                "custom": {}
+              },
+              "overrides": []
+            },
+            "format": "none",
+            "gauge": {
+              "maxValue": 100,
+              "minValue": 0,
+              "show": false,
+              "thresholdLabels": false,
+              "thresholdMarkers": true
+            },
+            "gridPos": {
+              "h": 2,
+              "w": 3,
+              "x": 9,
+              "y": 8
+            },
+            "id": 77,
+            "interval": null,
+            "links": [
+              {
+                "targetBlank": true,
+                "url": "[[kibana]],query:(language:lucene,query:'@log_stream:+%22host-inventory-syndicator-*%22+AND+message:+%22DB+Name\:+insights%22'),sort:!('@timestamp',desc))"
```

Gives a JSON validation error bc of an invalid escape character (after "Name"). Is that part even needed, though? If not I'd just simplify it to this:
```suggestion
                "url": "[[kibana]],query:(language:lucene,query:'@log_stream:+%22host-inventory-syndicator-*%22'),sort:!('@timestamp',desc))"
```